local-dev/sql_validate.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# validate postgres
+echo "sleeping for 5 seconds during postgres boot..."
+sleep 5
+export PGPASSWORD=wmpwd
+psql --username wmuser --host=postgres --port=5432 -d wm -c "SELECT VERSION();SELECT NOW()"
```

Comment:
If we make the above tweak to reuse the container:
```suggestion
PGPASSWORD=wmpwd psql --username wmuser -d wm -c "SELECT VERSION();SELECT NOW()"
```